pom.xml

```diff
@@ -556,10 +551,8 @@
           <version>3.12.0</version>
           <configuration>
             <!-- Exclude generated sources. -->
-            <excludePackageNames>
-              com.puppycrawl.tools.checkstyle.grammar.java:
-              com.puppycrawl.tools.checkstyle.grammar.javadoc:
-            </excludePackageNames>
+            <excludePackageNames>com.puppycrawl.tools.checkstyle.grammar.java:
```

Comment:
yes not nice yes, but who cares?

we have now a format for us now, to be straight forward.

I also would go with ned telling, "idk".

> I don't care where it goes, fine with me

- https://github.com/diffplug/spotless/pull/2687#discussion_r2436709418